src/pages/WalletsItemModeDisable/WalletsItemModeDisable.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+
+import React, { PureComponent } from 'react'
+import { t } from 'ttag'
+
+import { Button } from 'components/base'
+import { UserActionInfo } from 'components'
+import { walletsPlugin } from 'store/plugins'
+
+import walletsItemModeDisableStyle from './walletsItemModeDisable.m.scss'
+
+export type Props = {|
+  +walletId: string,
+|}
+
+export class WalletsItemModeDisable extends PureComponent<Props> {
+  handleDisable = () => {
```

Comment:
If it would be separated function instead method of class, then component would be stateless and we could make it memoized via `React.memo`.

Reply:
Difficult to imagine situation when this page will be re-rendered several time for user. So I think we will not have any benefits with memo.